source/brailleDisplayDrivers/handyTech.py

```diff
@@ -128,9 +129,9 @@ class Model(AutoPropertyObject):
 	#: @type: string
 	deviceId = None
 
-	#: A reference to the driver instance
+	#: A weak reference to the driver instance
 	#: @type; BrailleDisplayDriver
-	_display = None
+	_displayRef = None

```

Comment:
While some model attributes are retrieved from the model class rather than an instance of the class, this attribute is only used for instances. I think setting self._displayRef to the weakref in __init__ is enough and would personally just remove this definition.